master/internal/db/postgres_experiments.go

```diff
@@ -908,6 +908,8 @@ func (db *PgDB) DeleteExperiments(ctx context.Context, ids []int) error {
 		return errors.Wrapf(err, "error deleting validations for experiments %v", ids)
 	}
 
+	// Even with migration away from checkpoints_v1,
```

Comment:
wouldn't there be no checkpoints in checkpoints_v1?

Reply:
I figured they would all be migrated, or the user would've migrated back to a version where the code still deletes them